app/forms/create_profile_form.rb

```diff
@@ -0,0 +1,66 @@
+class CreateProfileForm
+  include Virtus.model
+  include ActiveModel::Model
+
+  attr :associated_user
+
+  attribute :name, String
+  attribute :tel, String
+  attribute :mobile, String
+  attribute :address, String
+  attribute :postcode, String
+  attribute :email, String
+  attribute :associated_user, String
+  attribute :password, String
+  attribute :password_confirmation, String
+
+  validates :name, :tel, :mobile, :address, :postcode, :email, presence: true
+  validates :password, :password_confirmation, presence: true, if: :associated_user?
+  validate :verify_unique_email
+  validate :matching_passwords
```

Comment:
Do we want validations defined here as well as in the models? We should pick one of the other.

Using https://github.com/apotonick/reform#validations-from-models would allow us to do both but I would prefer to keep the validations in 1 place. @pixeltrix ?